excel/beginner/charts/why-are-charts-useful.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Why are charts useful?
+
+---
+## Content
+
+A good example of why charts are useful is because they can be used to detect patterns and understand results in a more clear way.
+
+They are good for understanding large quantities of data with ease and speed.
+
+A single chart can easily represent thousands of data.
+
+Each type of chart also "tells a story".
+
+This is because upon creating a chart with large data, you can instantly see patterns and visualize the data and use it to determine any negative or positive information regarding the data.

```

Comment:
I'd simplify this sentence a little bit.

Maybe:

```suggestion
This is because visualizations can quickly demonstrate patterns and allow us to draw conclusions and determine any negative or positive outcomes about our data.
```